src/lib/operators/projection.cpp

```diff
@@ -12,6 +12,9 @@
 #include "expression/expression_utils.hpp"
 #include "expression/pqp_column_expression.hpp"
 #include "expression/value_expression.hpp"
+#include "hyrise.hpp"
```

Comment:
(Random line)
Any idea what causes the performance changes for single-threaded execution? I'm happy we have them, I just wonder why.

Reply:
Unfortunately, I think it's (quite) heavy noise. Q2 and 17 have a projection runtime share of 0%. 
Sometimes, Q18 loses, sometimes it wins. Also 0% projection runtime share.